docker/examples/.env

```diff
@@ -0,0 +1,3 @@
+# replace with path containing /IWAD and /WADS
+DOOM_BASE=~/Documents/GZDoom
```

Comment:
I'd probably make the example `DOOM_BASE` be something like `~/wads` or `~/Doom/wads`.